angularjs/app/app.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+// Declare app level module which depends on views, and components
+
+var app = angular.module('myApp', [
+  'ngRoute',
+  'myApp.view1',
+  'myApp.view2',
```

Comment:
What do you think about simplify this example by having just a single view? So it's easy for customers to find the Wootric code.